sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/RectangleEditor.java

```diff
@@ -213,8 +213,11 @@ private void adjustWidth(ControlPoint oCP, IGeoPosition oLatLon) {
         Log.d(TAG, oCP.getCPType().toString() + currentWidth + " " + currentBearing);
         recompute(this.oFeature.getPosition());
         this.oFeature.setWidth(currentWidth);
-        this.oFeature.apply();
-
+        try {
+            storageManager.apply(mapInstance, oFeature, null);
+        } catch (EMP_Exception ex) {
+            Log.e(TAG, "Apply failed.", ex);
```

Comment:
What actually happens when apply fails? Looks like we just log and move on.

Reply:
This is in the draw editors.  If the change fails to apply, the user will see it.